### DISCUSSION TITLE
Anyway to report .jar in-used by app on a  system ?

### DISCUSSION THREAD

**Discussion by tjyang** [fox-it/log4j-finder#67] [Ideas]
I found there are many system paths storing old .jar files  from "log4j-finder /qn" scan.
But I am sure they are not in used and loaded by app.

Is there a better way to only report old .jar are currently used by system ?

**Comment by yunzheng** [comment]
I'm not sure how to determine if a jar is used or not when it's on the disk. I have seen other script just scanning the running processes (eg via /proc/ filesystem), but this also has some drawbacks.

**Comment by tjyang** [comment]
someone give script to scan running processes, like following.

```
log4j_result=`ls -la /proc/*/fd/ 2>/dev/null|grep -E "log4j-core"|cut -d '>' -f2|sort -u|grep -v rc2.jar` 2>/dev/null
```
- How about report *.jar  or java running processes  under /proc to simply the work as a start.
